components/NavBar/NavBar.styled.ts

```diff
@@ -209,14 +215,13 @@ export const DesktopOnlySection = styled.section`
 `;
 
 export const DesktopNavLink = styled.a<NavLinkProps>`
-  font-family: GilroyMedium;
-  color: ${({ isActive }) => (isActive ? '#0e103c' : '#7578b5')};
+  color: ${({ isActive }) => (isActive ? '#0e103c' : '#0e103c')};
```

Comment:
These are the same colors?

Reply:
Switched the old colors (new mocks show the dark color as the inactive link)!